src/main/java/com/habibank/controller/AuthController.java

```diff
@@ -1,135 +1,135 @@
-package com.habibank.controller;
+// package com.habibank.controller;
```

Comment:
all of these lines are commented out - can we delete the file if it is not needed?